custom_components/sensor/opensprinkler.py

```diff
@@ -40,24 +40,17 @@
 
 utc_tz = pytz.timezone('UTC')
 
-# sensor_type [ description, unit, icon ]
+# sensor_type [ description, unit, icon, name ]
 SENSOR_TYPES = {
-    'opensprinkler_rain': ['os_rain', None, 'mdi:weather-rainy'],
-    'opensprinkler_lastrun': ['os_lastrun', None, 'mdi:history'],
-    'opensprinkler_pump': ['os_pump', None, 'mdi:water-pump'],
-    'opensprinkler_station': ['os_station', None, 'mdi:leaf'],
-    'opensprinkler_schedule': ['os_schedule', None, 'mdi:calendar-clock']
-}
-SENSOR_TYPES_DEFAULT = {
```

Comment:
Not sure if you still wanted to have separate for these, but it seemed to be redundant